lib/lin.ml

```diff
@@ -31,7 +31,9 @@ module type CmdSpec = sig
   (** [run c t] should interpret the command [c] over the system under test [t] (typically side-effecting). *)
 end
 
-module Make(Spec : CmdSpec) (*: StmTest *)
+(** A functor to create Domain and Thread test setups.
+    We use it below, but it can also be used independently *)
+module MakeDomThr(Spec : CmdSpec)
```

Comment:
I agree!
- one option is to rename it to something better
- another option is to just exclude it from the public interface, since `Make` is now a functor building all three interpretations (Domain, Thread, and Effect). 

(It just happens to be handy to reuse this first functor to create the last interpretation)